### PR TITLE
`slack:notifier` : Exclude prestashop-specs from PR sent to devs

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | `slack:notifier` : Exclude prestashop-specs from PR sent to devs
| Type?             | improvement

### CODE REVIEW

File: src/App/Service/Github.php [hunk 1]
```diff
@@ -273,7 +273,19 @@ public function isPRValid(array $pullRequest, Filters $filters): bool
             return false;
         }
 
-        // Filter Repository PRovaite
+        // Filter Repository Name
+        if ($filters->hasFilter(Filters::FILTER_REPOSITORY_NAME)) {
+            if ($filters->isFilterIncluded(Filters::FILTER_REPOSITORY_NAME)
+                && !in_array($pullRequest['repository']['name'], $filters->getFilterData(Filters::FILTER_REPOSITORY_PRIVATE), true)) {
```
Comment (PierreRambaud) [inline src/App/Service/Github.php:279]:
```suggestion
                && !in_array($pullRequest['repository']['name'], $filters->getFilterData(Filters::FILTER_REPOSITORY_NAME), true)) {
```

File: src/App/Service/Github.php [hunk 2]
```diff
@@ -273,7 +273,19 @@ public function isPRValid(array $pullRequest, Filters $filters): bool
...
+                return false;
+            }
+            if (!$filters->isFilterIncluded(Filters::FILTER_REPOSITORY_NAME)
+                && in_array($pullRequest['repository']['name'], $filters->getFilterData(Filters::FILTER_REPOSITORY_PRIVATE), true)) {
```
Comment (PierreRambaud) [inline src/App/Service/Github.php:283]:
```suggestion
                && in_array($pullRequest['repository']['name'], $filters->getFilterData(Filters::FILTER_REPOSITORY_NAME), true)) {
```

But I don't get the double check.
Only the condition is needed imho

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Few comments

File: src/App/Service/Github.php (hunk 2 above)
Comment (Progi1984) [reply src/App/Service/Github.php]: Fixed

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks you @Progi1984

**Pull request merged**